view/statistics-time-per-person.js

```diff
@@ -39,14 +39,25 @@ var getRowResult = function (rowData, label) {
 	return result;
 };
 
+var processingStepsMetaFrontDeskFilter = function (processingStepsMeta) {
+	var processingStepsMetaWithoutFrontDesk = {};
+	Object.keys(processingStepsMeta).forEach(function (key) {
+		if (key !== 'frontDesk') {
```

Comment:
use this module `require('es5-ext/string/#/starts-with')` to check for `frontDesk`... there are steps which are treated as frontDesk and have `frontDesk` only as prefix... we want to filter them as well.